CHANGELOG.md

```diff
@@ -1,5 +1,64 @@
 # CHANGELOG
 
+## 2.0.0 (2022-11-14)
+
+MLflow 2.0.0 includes several breaking changes from the 1.x API, additional major features and improvements
+
+Breaking Changes:
+
+- [Recipes] Renamed `mlflow.pipelines` to `mlflow.recipes`
+- [Tracking] Remove deprecated `list` APIs for experiments, models, and runs from Python, Java, JS, and R APIs (#6785, #6786, #6787, #6788, #6800, #6868, @dbczumar)
+- [Tracking] Remove deprecated `runs` data from `get_experiment` REST API response (#6541, #6524 @dbczumar)
+- [Tracking] Remove deprecated `MLflowClient.download_artifacts` API (#6537, @WeichenXu123)
+- [Tracking] Add support for experiment creation resolution and validation for `MLFLOW_EXPERIMENT_NAME` and `MLFLOW_EXPERIMENT_ID` environment variables (#6674, @BenWilson2)
+- [Tracking] Modify tracking server configuration to use `--serve-artifacts` by default (#6502, @harupy)
+- [Tracking] Introduce random `experiment_id` generation for `filestore` tracking server backend to enable threadsafe concurrency (#7070, @BenWilson2)
+- [Tracking] Remove `dataset_name` and `on_data_{name | hash}` suffixes from `mlflow.evaluate` metric keys (#7042, @harupy)
+- [Models] Remove deprecated `mlflow.keras` module and enable `keras` support within `tensorflow` flavor (#6530, @WeichenXu123)
+- [Models] Remove deprecated `mlflow.sklearn.eval_and_log_metrics` API in favor of `mlflow.evaluate` API (#6520, @dbczumar)
+- [Models] Change the default environment configuration for `spark_udf` loaded models to use `local` (#6640, @WeichenXu123)
+- [Models] Change default to `virtualenv` instead of `conda` for Models environment creation (#6459, @harupy)
+- [Models] Remove support for `pyfunc` submission to `mlflow.evaluate` (#6670, @harupy)
+- [Models] Modify `PyfuncModel` spec to support `conda` and `virtualenv` subfields (#6684, @harupy)
+- [Scoring] Introduce strict format indicators for input datatypes for the scoring API and modify the output structure of returned values (#6575, @tomasatdatabricks)
+- [Scoring] Remove public APIs for `mlflow.sagemaker.deploy()` and `mlflow.sagemaker.delete()` (#6650, @dbczumar)
+- [Scoring] Rename input argument `df` to `inputs` in `mlflow.deployments.predict()` method (#6681, @BenWilson2)
+- [Scoring] Replace the `--no-conda` cli option argument for native serving with `--env-manager='local'` (#6501, @harupy)
+- [Registry / Tracking] Remove `/preview` routes for Tracking and Model Registry REST APIs (#6667, @harupy)
+- [Projects] Remove the `use_conda` argument from `run` cli command for MLflow Projects (#6654, @harupy)
+- [Projects] Introduce support for `virtualenv` within ML Projects and set as default environment (#6489, @harupy)
```

Comment:
Virtualenv support was already added in 1.x, this just makes it the default. Let's consolidate this with other virtualenv breaking changes and move to the top (see https://github.com/mlflow/mlflow/pull/7324/files#r1021181151)